Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,16 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    system "pkill postgres || true"
+
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version_suffix}"
```

Comment:
Seem to be hitting the same Ruby choke that we hit [earlier](https://github.com/Homebrew/homebrew/pull/35577#discussion_r22505181) :confused:, unless I'm screwing something up?

```
  pg_version = (var/"postgres/PG_VERSION").read.chomp
  plist_options :manual => "postgres -D #{HOMEBREW_PREFIX}/var/postgres-#{pg_version}"
```